eslint.config.js

```diff
@@ -0,0 +1,14 @@
+import eslintCommonConfig from "./eslint.common.config.js";
+
+export default [
+	...eslintCommonConfig, // Load common ESLint config
+	{
+		files: ["bin/ui5.cjs"],
+
+		languageOptions: {
+			globals: {},
+			ecmaVersion: 2022,
```

Comment:
Is it possible to set this back to `2020` like it was before? Or does it cause parsing errors?